spec/index.bs

```diff
@@ -825,7 +825,7 @@ interface GPUDevice : GPUObjectBase {
     GPUMappedBuffer createBufferMapped(GPUBufferDescriptor descriptor);
     Promise<GPUMappedBuffer> createBufferMappedAsync(GPUBufferDescriptor descriptor);
     GPUTexture createTexture(GPUTextureDescriptor descriptor);
-    GPUSampler createSampler(GPUSamplerDescriptor descriptor);
+    GPUSampler createSampler(optional GPUSamplerDescriptor descriptor);
```

Comment:
I don't think sampler descriptor can be optional: there are too many non-obvious choices to make

Reply:
GPUSamplerDescriptor has default values for every choice. See below. Which are the ones that shouldn't be default?

```idl
dictionary GPUSamplerDescriptor : GPUObjectDescriptorBase {
    GPUAddressMode addressModeU = "clamp-to-edge";
    GPUAddressMode addressModeV = "clamp-to-edge";
    GPUAddressMode addressModeW = "clamp-to-edge";
    GPUFilterMode magFilter = "nearest";
    GPUFilterMode minFilter = "nearest";
    GPUFilterMode mipmapFilter = "nearest";
    float lodMinClamp = 0;
    float lodMaxClamp = 0xffffffff; // TODO: What should this be? Was Number.MAX_VALUE.
    GPUCompareFunction compare = "never";
};
```

Reply:
These defaults were introduced in #73 about 10 months ago, way before the "default" topic came into the consideration of the group. I suggest this PR postponed until the group discusses these wanna-be-defaults (again).

Reply:
How about having non default values for`magFilter`, `minFilter` and `mipmapFilter`?
We can keep the other defaults. WDYT?

Reply:
why would `addressMode?` default to anything?